bundles/org.eclipse.swt/Readme.macOS.md

```diff
@@ -82,13 +82,30 @@ More information on how to use the generator is available [here](https://www.ecl
 this information into this readme.
 
 
-Compiling native libs
----------------------
+## Building Natives
 
-This is done from within the binaries project.
+### Building 64bit natives for Cocoa
 
-    cd <eclipse.platform.swt>/binaries/org.eclipse.swt.cocoa.macosx.x86_64
-    mvn clean package -Dnative=cocoa.macosx.x86_64
+You need to install the following on your system:
 
+* TBD
+* Java 17 JDK
```

Comment:
@HeikoKlare or @Phillipus can you name the tools that have to be installed on a mac in order to be able to build SWT locally for mac?

Reply:
@HannesWell i just have Java and Maven.

Reply:
So on Mac everything to compile C code is already installed out of the box?

Reply:
Good point. I have Apple's `Command Line Tools` installed which is kind of a lightweight version of `Xcode`. So I would add that one of these needs to be installed as a requirement.

Reply:
I can confirm that `Command Line Tools` are sufficient. I am not sure whether it is worth to note that if you want to build the binaries with gcc, you need to install that on MacOS by hand. By default `cc` will execute a `clang` compiler on MacOS.